### ISSUE TITLE
Cover Block placeholder does not adjust text or icon colors for Dark Themes in editor

### ISSUE THREAD

**Issue by getdave** [WordPress/gutenberg#14306]
The Cover Block doesn't adjust it's text or icon color in [dark theme mode](https://wordpress.org/gutenberg/handbook/designers-developers/developers/themes/theme-support/#dark-backgrounds):

**To Reproduce**
1. Install a Theme which supports [`dark-editor-style`](https://wordpress.org/gutenberg/handbook/designers-developers/developers/themes/theme-support/#dark-backgrounds)
2. Open the Editor
3. Insert the `Cover` Block
4. Notice that the text and icon are dark against a dark background

**Expected behavior**
Text should be clearly legible and provide adequate contrast with dark background.

**Screenshots**

![screen shot 2019-03-07 at 09 17 27](https://user-images.githubusercontent.com/444434/53949137-53c81c80-40c1-11e9-987f-df7151349291.png)

**Additional context**

See also https://github.com/WordPress/gutenberg/pull/14241#issuecomment-470449343

**Comment by kjellr** [comment]
@getdave can you confirm this still exists? Testing in `master`, with a dark theme, I'm seeing this today: 

![Screen Shot 2019-03-18 at 1 52 25 PM](https://user-images.githubusercontent.com/1202812/54551795-244cc600-4985-11e9-9fae-f4f7cec9e5f4.png)

**Comment by getdave** [comment]
Seems to be fixed to me.
